rpc_jobs/rpc_leapfrog.yml

```diff
@@ -0,0 +1,194 @@
+- project:
+    name: 'RPC-Leapfrog-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    series:
+      - ktoN:
+          branch: kilo
+          leapfrog_branch: master
+          branches: "leapfrog.*"
+    image:
+      - trusty:
+          IMAGE: "Ubuntu 14.04.5 LTS prepared for RPC deployment"
+    context:
+      - oneshot
+      - interrupted
+    exclude:
+      # We shouldn't test the interrupt/resume on PRs. Only on periodics.
+      - context: interrupted
+        ztrigger: pr
+    ztrigger:
+      - pr:
+          CRON: ""
+          TRIGGER_USER_VARS: "maas_use_api: false"
+      - periodic:
+          branches: "do_not_build_on_pr"
+          NUM_TO_KEEP: 10
+    jobs:
+      - 'RPC-Leapfrog_{series}-{image}-{context}-{ztrigger}'
+
+- job-template:
```

Comment:
We need a default for `NUM_TO_KEEP`, ie. https://github.com/rcbops/rpc-gating/blob/master/rpc_jobs/rpc_aio.yml#L103.